test/Sentry.Unity.Editor.Tests/SentryScriptableObjectTests.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.IO;
+using NUnit.Framework;
+using UnityEditor;
+
+namespace Sentry.Unity.Editor.Tests
+{
+    public class SentryScriptableObjectTests
+    {
+        private string _tempPath = null!; // Assigned in Setup
+
+        [SetUp]
+        public void Setup() => _tempPath = Path.Combine("Assets", Guid.NewGuid().ToString(), "TestOptions.asset");
+
+        [TearDown]
+        public void TearDown() => AssetDatabase.DeleteAsset(Path.GetDirectoryName(_tempPath));
+
+        [Test]
+        public void Load_ScriptableSentryUnityOptionsDoesNotExist_ReturnsNewObject()
+        {
+            SentryScriptableObject.Load<ScriptableSentryUnityOptions>(_tempPath);
```

Comment:
Best to take this temp path here, so there's no chance of another test (that ran before on concurrently) to share that resource. Maybe the file already existed?

Reply:
The `[Setup]` creates a new path containing a Guid. Does that suffice?